playbooks/roles/filebeat/templates/filebeat.yml.j2

```diff
@@ -0,0 +1,215 @@
+filebeat:
+  prospectors:
+    -
+      paths:
+        - "{{ nginx_log_dir }}/kibana.access.log"
+      fields:
+        nginx_site: kibana
+        openedx_log_format: nginx_access
```

Comment:
The other link called this "type"; let's check a few more references for any momentum. if they all say "type", let's use that. Else, re-evaluate.